docs/docs/dev_docs/tutorials/testing.md

```diff
@@ -137,7 +137,7 @@ Private state in the Aztec Network is represented via sets of [private notes](..
 
 #include_code value-note-def yarn-project/aztec-nr/value-note/src/value_note.nr rust
 
-We can query the RPC server for all notes encrypted for a given user in a contract slot. For this example, we'll get all notes encrypted for the `owner` user that are stored on the token contract address and on the slot we calculated earlier. To calculate the actual balance, we extract the `value` of each note, which is the first element, and sum them up.
+We can query the Private eXecution Service (PXE) for all notes encrypted for a given user in a contract slot. For this example, we'll get all notes encrypted for the `owner` user that are stored on the token contract address and on the slot we calculated earlier. To calculate the actual balance, we extract the `value` of each note, which is the first element, and sum them up.
```

Comment:
Should this be called "Private eXecution Environment" or do you want to keep it as "Service" in order to differentiate it from the "PXE client"?

Reply:
This is a mistake. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/2574/commits/3c5addd7d64b0d906af8a764d92ad56e4152ecfc

Great catch 👍